client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -913,7 +913,9 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow, stepName } ) => {
 			<StepContainer
 				stepName={ STEP_NAME }
 				stepContent={ stepContent }
-				hideSkip
+				hideSkip={ ! isGoalsHoldout }
+				skipLabelText={ translate( 'Skip setup' ) }
```

Comment:
It would be better to keep `Skip` to align with other steps. What do you think? cc @fditrapani

Reply:
Thanks for bringing this up @arthur791004. There have been discussions to have a single persistent `Skip setup` link on all the onboarding steps that behaves the same across the board. We'll tackle the other steps separately and can then merge this in together with the others when they're ready.